src/components/SavedProjectPlan/StandardCard.jsx

```diff
@@ -0,0 +1,44 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Flex,
+  Heading,
+  Image,
+} from "@chakra-ui/react";
+const StandardCard = () => {
+  return (
+    <Card flex="1">
+      <CardHeader>
+        <Heading size="lg">Standard 1</Heading>
+      </CardHeader>
+      <CardBody marginLeft="3" marginRight="3">
+        <Flex>
+          <Box flex="1">
+            <Image
+              src="../../../public/static/ShoppingCartImg.png"
```

Comment:
next.js has this thing where the public folder is already the root for all `src` props. So you can just do something like this `<Image src="/static/[imageName].png">`

Reply:
next has this thing where the public folder is already the root for all `src` props. So you can just do something like this <Image src="/static/[imageName].png">